.ambient-package-update/metadata.py

```diff
@@ -0,0 +1,44 @@
+from ambient_package_update.metadata.author import PackageAuthor
+from ambient_package_update.metadata.constants import (
+    DEV_DEPENDENCIES,
+    LICENSE_MIT,
+    SUPPORTED_DJANGO_VERSIONS,
+    SUPPORTED_PYTHON_VERSIONS,
+)
+from ambient_package_update.metadata.maintainer import PackageMaintainer
+from ambient_package_update.metadata.package import PackageMetadata
+from ambient_package_update.metadata.readme import ReadmeContent
```

Comment:
This feels overly verbose and would probably benefit from being something that could be imported as a module.

Reply:
I was thinking more like how django suggests you handle models, `from django.db import models`

Reply:
Sounds reasonable. Stick to known pattern. 👌